miniapps/meshing/check-metric.cpp

```diff
@@ -0,0 +1,240 @@
+// Copyright (c) 2010-2022, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+//
+// Checks evaluation / 1st derivative / 2nd derivative for TMOP metrics.
+//
+
+#include "mfem.hpp"
+#include <iostream>
+
+using namespace mfem;
+using namespace std;
+
+int main(int argc, char *argv[])
+{
+   int metric_id = 1;
```

Comment:
Let's change the default `metric_id` to 2 since required methods are not yet implemented for metric 1.